akka-typed/src/main/scala/akka/typed/internal/EventStreamImpl.scala

```diff
@@ -0,0 +1,332 @@
+
+    def sendSystem(signal: SystemMessage): Unit = ()
+
+    @throws(classOf[java.io.ObjectStreamException])
+    protected def writeReplace(): AnyRef = serializedStandardOutLogger
+  }
+
+  private val serializedStandardOutLogger = new SerializedStandardOutLogger
+
+  /**
+   * INTERNAL API
+   */
+  @SerialVersionUID(1L) class SerializedStandardOutLogger extends Serializable {
+    @throws(classOf[java.io.ObjectStreamException])
+    private def readResolve(): AnyRef = StandardOutLogger
+  }
+
+  val StandardOutLogger = new StandardOutLogger
+
+  val UnhandledMessageForwarder = Static[a.UnhandledMessage] {
+    case a.UnhandledMessage(msg, sender, rcp) ⇒
+      publish(Debug(rcp.path.toString, rcp.getClass, "unhandled message from " + sender + ": " + msg))
+  }
+
+  /**
+   * Internal Akka use only
+   */
+  private[akka] def startStdoutLogger(settings: Settings) {
+    setUpStdoutLogger(settings)
+    publish(Debug(simpleName(this), this.getClass, "StandardOutLogger started"))
+  }
+
+  /**
+   * Internal Akka use only
+   */
+  private[akka] def startDefaultLoggers(system: ActorSystemImpl[Nothing]) {
+    val logName = simpleName(this) + "(" + system + ")"
+    val level = levelFor(system.settings.untyped.LogLevel) getOrElse {
+      // only log initialization errors directly with StandardOutLogger.print
+      StandardOutLogger.print(Error(new LoggerException, logName, this.getClass, "unknown akka.loglevel " + system.settings.untyped.LogLevel))
+      ErrorLevel
+    }
+    try {
+      val defaultLoggers = system.settings.Loggers match {
+        case Nil     ⇒ classOf[DefaultLogger].getName :: Nil
+        case loggers ⇒ loggers
+      }
+      val myloggers =
+        for {
+          loggerName ← defaultLoggers
+          if loggerName != StandardOutLogger.getClass.getName
+        } yield {
+          system.dynamicAccess.getClassFor[Logger](loggerName).map(addLogger(system, _, level, logName))
+            .recover({
+              case e ⇒ throw new akka.ConfigurationException(
+                "Logger specified in config can't be loaded [" + loggerName +
+                  "] due to [" + e.toString + "]", e)
+            }).get
+        }
+      guard.withGuard {
+        loggers = myloggers
+        _logLevel = level
+      }
+      if (system.settings.untyped.DebugUnhandledMessage)
+        subscribe(
+          ActorRef(
+            system.systemActorOf(UnhandledMessageForwarder, "UnhandledMessageForwarder")),
+          classOf[a.UnhandledMessage])
+      publish(Debug(logName, this.getClass, "Default Loggers started"))
+      if (!(defaultLoggers contains StandardOutLogger.getClass.getName)) {
+        unsubscribe(StandardOutLogger)
+      }
+    } catch {
+      case e: Exception ⇒
+        System.err.println("error while starting up loggers")
+        e.printStackTrace()
+        throw new akka.ConfigurationException("Could not start logger due to [" + e.toString + "]")
+    }
+  }
+
+  /**
+   * Internal Akka use only
```

Comment:
this should all go away, I only forgot this one: within the `akka.typed.internal` package EVERYTHING is internal by definition